cucumber/kubernetes/features/http_proxy.feature

```diff
@@ -0,0 +1,8 @@
+Feature: A permitted Conjur host can authenticate with a valid resource restrictions
+  that is defined in the id and the kubernetes host can be reached through a
+  http_proxy
+
+  @http_proxy
```

Comment:
I'm guessing there must be some sort of behind-the-scenes magic connecting this tag to the `ci/authn-k8s/dev/dev_conjur_http_proxy.template.yaml`?

It would probably be worth adding a small comment above this test explaining what it does, and perhaps link to the template file for reference in the future.

Reply:
comment added :)